recipes/devel/m4.yaml

```diff
@@ -15,3 +15,5 @@ buildScript: |
 packageScript: |
     autotoolsPackageTgt
 
+provideTools:
+    m4: "usr/bin"
```

Comment:
Hmm, so far `m4` is a basement build prerequisite. It has to be installed on the host and is available in the sandbox for that reason. Is this not sufficient?

If we provide it as tool we may want to do this consistently. That is, also drop it from the sandbox...